src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="stars stars--1">
```